airbyte-integrations/connectors/destination-redshift/src/main/resources/spec.json

```diff
@@ -45,6 +45,54 @@
         "description": "Whether or not to normalize the data in the destination. See <a href=\"https://docs.airbyte.io/architecture/basic-normalization\">basic normalization</a> for more details.",
         "title": "Basic Normalization",
         "examples": [true, false]
+      },
```

Comment:
Should these new fields all be marked as required?

Reply:
Nope. We want to leave the option of not using the more efficient copy strategy to users since it can take a while to set up the buckets + get the right keys. I tried to form this into an object and ran into spec.json rendering issues.